src/client.js

```diff
@@ -137,6 +137,16 @@ Client.prototype.connect = function(args) {
 
 	var nick = args.nick || "lounge-user";
 	var webirc = null;
+	var channels = [];
+
+	if (args.join) {
+		var join = args.join.replace(/\,/g, " ").split(/\s+/g);
+		join.forEach(function(chan) {
+			channels.push(new Chan({
+				name: chan
+			}));
+		});
```

Comment:
Nevermind, got ruled out in priv by @xPaw, who plans to use the `channels` variable in a further PR and cannot blindly overwrite it.